pkg/sfu/forwarder.go

```diff
@@ -1477,9 +1477,10 @@ func (f *Forwarder) getTranslationParamsVideo(extPkt *buffer.ExtPacket, layer in
 	if f.ddLayerSelector != nil {
 		if selected := f.ddLayerSelector.Select(extPkt, tp); !selected {
 			tp.shouldDrop = true
+			f.rtpMunger.UpdateAndGetSnTs(extPkt) // call to update highest incoming sequence number and other internal structures
```

Comment:
This is required to set up RTP munger state before indicating a drop. Without this, the receiving side saw a bunch of gaps (packets lost) and was NACKing a lot. This is not a new thing. Happened with v1.3.0 code also. It's always been there.